application/models/organization_test.go

```diff
@@ -197,3 +184,80 @@ func TestValidateOrganization(t *testing.T) {
 		})
 	}
 }
+
+func TestOrganizationFindByDomain(t *testing.T) {
+	org, orgDomain := createOrgFixtures(t)
+
+	type args struct {
+		domain string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    Organization
+		wantErr bool
+	}{
+		{
+			name: "found",
+			args: args{orgDomain.Domain},
+			want: org,
+		},
+		{
+			name:    "empty string",
+			args:    args{""},
+			wantErr: true,
+		},
+		{
+			name:    "near match",
+			args:    args{"example.com"},
+			wantErr: true,
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			got, err := OrganizationFindByDomain(test.args.domain)
+			if test.wantErr {
+				if err == nil {
+					t.Errorf("Expected an error, but did not get one")
+				}
+			} else {
+				if err != nil {
+					t.Errorf("OrganizationFindByDomain() returned an error: %v", err)
+				} else if got.Uuid != test.want.Uuid {
+					t.Errorf("found %v, expected %v", got, test.want)
+				}
+			}
+		})
+	}
+
+	// delete org fixture and org domain by cascading delete
```

Comment:
There is a `resetTables` functions in testutils.go already, do you want to use one of those here?

Reply:
I did this before @fillup created `resetTables()`. This is a bit simpler in that it doesn't need to clear out the `users` table.